src/librustdoc/html/static/js/rustdoc.d.ts

```diff
@@ -467,7 +467,7 @@ declare namespace rustdoc {
      * Provied by generated `trait.impl` files.
      */
     type Implementors = {
-        [key: string]: Array<[string, number, Array<string>]>
+        [key: string]: Array<[string, number, number, Array<string>]>
```

Comment:
```suggestion
        [key: string]: Array<[string, number, 0|1, Array<string>]>
```

we can be a bit more specific here.

also, it would be nice to expand the comment to mention what the meaning of each field is.